### PR TITLE
Add reference to PHP client library

### PR DESCRIPTION
**Pull request by bracki**
This adds a link to our PHP client [library](https://github.com/Jimdo/prometheus_client_php).

### CODE REVIEW

File: content/docs/instrumenting/clientlibs.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Unofficial third-party client libraries:
 * [Lua](https://github.com/knyar/nginx-lua-prometheus) for Nginx
 * [.NET / C#](https://github.com/andrasm/prometheus-net)
 * [Node.js](https://github.com/siimon/prom-client)
+* [PHP] (https://github.com/Jimdo/prometheus_client_php)
```
Comment (beorn7) [inline content/docs/instrumenting/clientlibs.md]: The GitHub markdown renderer is fine with the blank in between here, but could you remove it for consistency?

**Comment by beorn7** [comment]
@brian-brazil @juliusv What's the state of the discussion here? I didn't follow it closely. Is @bracki `s library the one to link? What about https://github.com/Lazyshot/prometheus-php ?

**Comment by brian-brazil** [comment]
As far as I can see Lazyshot's still can't work as it overwrites stats on each request.

**Comment by beorn7** [comment]
Which leaves the question if we should link @bracki 's library.

**Comment by bracki** [comment]
Just to clarify: This client library doesn't lose stats over multiple requests.

**Comment by beorn7** [comment]
@brian-brazil At this point, I'd simply merge this PR. Do you have any objections?

**Comment by brian-brazil** [comment]
Let's add it.

**Comment by juliusv** [comment]
Good, merging - thanks :)

**Pull request merged**